shell/platform/android/io/flutter/embedding/android/FlutterActivityAndFragmentDelegate.java

```diff
@@ -496,6 +544,10 @@ void onDestroyView() {
     Log.v(TAG, "onDestroyView()");
     ensureAlive();
 
+    if (activePreDrawListener != null) {
+      flutterView.getViewTreeObserver().removeOnPreDrawListener(activePreDrawListener);
```

Comment:
This method throws if `flutterView.getViewTreeObserver().isAlive()` returns `false`: https://developer.android.com/reference/android/view/ViewTreeObserver#removeOnPreDrawListener(android.view.ViewTreeObserver.OnPreDrawListener)